network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -1,71 +1,100 @@
-    <meta property="og:image" content="{{request.scheme}}://{{request.get_host}}/_images/buyers-guide/share.jpg">
-    <meta property="og:image:type" content="image/jpg">
-    <meta property="og:image:width" content="1200">
-    <meta property="og:image:height" content="628">
-    <meta property="og:url" content="{{request.scheme}}://{{request.get_host}}{{request.path}}">
-    <link rel="stylesheet" href="/_css/buyers-guide.compiled.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700">
-    <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/_images/favicons/apple-touch-icon-180x180@2x.png">
-    <link rel="icon" type="image/png" sizes="196x196" href="/_images/favicons/favicon-196x196@2x.png">
-    <link rel="shortcut icon" href="/_images/favicons/favicon.ico">
-    <title>{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - *privacy not included{% endif %}</title>
-    <script>
-      var modernBrowser = (
-        'fetch' in window &&
-        'assign' in Object
-      );
-
-      if ( !modernBrowser ) {
-        var scriptElement = document.createElement('script');
-
-        scriptElement.async = false;
-        scriptElement.src = '/_js/polyfills.compiled.js';
-        document.head.appendChild(scriptElement);
-      }
-    </script>
-  </head>
-  <body class="pni" id="pni-{% block body_id %}{% endblock %}">
-    <script src="/_js/bg-main.compiled.js" async defer></script>
-
-    {% include "./primary_nav.html" %}
-    <div class="underneath-screen-overlay">
-      <div class="category-nav">
-        <div class="container">
-          <div class="row">
-            <div class="col">
-              {% url 'buyersguide-home' as home_url %}
-              <a class="{% bg_active_nav request.get_full_path home_url %}" href="{{ home_url }}">Home</a>
-              {% category_nav request.get_full_path category categories %}
-              {% url 'about-view' as about_url %}
-              <a class="{% bg_active_nav request.get_full_path about_url %}" href="{{ about_url }}">About the Guide</a>
-            </div>
+
+{% block page_title %}{% if page_title %}{{ page_title }}{% else %}Mozilla - *privacy not included{% endif %}{% endblock %}
+
+
+{% block ga_identifier %}
+  <!-- <meta name="google-site-verification" content="..."> -->
+  <meta name="ga-identifier" content="UA-87658599-6">
+{% endblock %}
+
+
+{% block social_metadata %}
+  <meta property='og:type' content='website'>
+  <meta property='og:locale' content='en-US'>
+  <meta property="og:title" content="*Privacy Not Included: A Buyer’s Guide for Connected Products">
+{% endblock %}
+
+
+{% block wagtail_metadata %}
+  <meta property="og:description" content="89 connected products rated on their safety and security">
```

Comment:
Is it still 89 now that we only show recently reviewed products?

Reply:
this should now be 75